### PR TITLE
Add documention on module generation using Content Builder tool

### LINKED ISSUE #423: Better documentation how this collection is generated
##### SUMMARY
I don't understand how this collection is generated. Is this documented? [This](https://github.com/ansible-community/ansible.content_builder#5-vmwarevmware_rest) doesn't really help...

It would be really helpful to have a complete example how to create a module from the OpenAPI / Swagger specification. Maybe this exists and I just didn't find it. In this case please point me to it.

If there's no example, it would be really helpful if you could create an example how to create a module from an OpenAPI / Swagger specification.

BTW, it looks like you're depending on `https://github.com/goneri/ansible-collection-goneri.utils.git`. I think this is a problem... nothing against @goneri but we shouldn't rely on repos owned by a single person.

##### ISSUE TYPE
- Documentation Report

##### ANSIBLE VERSION

cc @jillr

### PR DESCRIPTION
**Pull request by GomathiselviS**
##### SUMMARY

Fixes #423 

##### ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- New Module Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Review by mariolenz — APPROVED** [review]
I'm not sure yet if this really answers all my questions, but it's a start and LGTM.

Should we merge it... release early, release often?

**Comment by GomathiselviS** [comment]
> I'm not sure yet if this really answers all my questions, but it's a start and LGTM.
> 
> Should we merge it... release early, release often?

Hi @mariolenz 
I have an example `MANIFEST.yaml` file and a command that can be used to generate a module from the OpenAPI/Swagger specification in the document. The repository also has sample specification files and modules.yaml that you can use. 

To get started, you can install the `ansible.content_builder` repository and use the `build.yaml` and `MANIFEST.yaml` files to generate the collection. If you encounter any issues, let me know and I'll do my best to assist you and update the doc.

Regarding the release, I would check with @jillr

File: development.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# VMWare
```
Comment (hakbailey) [inline development.md:1]: This all makes sense to me and I was able to follow it to generate the modules locally. I did notice that the generated modules didn't pass black linting locally. Do we want to include instructions to reformat the modules with black after they are generated?

Comment (hakbailey) [reply development.md:1]: Much better!

**Review by hakbailey — APPROVED** [review]
I wasn't able to try the "Refresh the RETURN Block" section, but everything else worked as documented.

**Comment by mariolenz** [comment]
I also had to install `boto3`, `baron`, `redbaron` and `ruamel.yaml`.

**Pull request merged**